src/Jackett.Common/Definitions/xbytes2.yml

```diff
@@ -143,6 +156,18 @@ search:
           args: ["RUS", "Russian"]
         - name: replace
           args: ["DUAL", "Spanish English"]
+        - name: re_replace
+          args: ["MHD", "MHD.BDRip"] # Fix for Radarr
+        - name: re_replace
+          args: ["WEB.DL", "WEBDL"] # Fix for Radarr
+        - name: re_replace
+          args: ["WEBDL.MHD.BDRip", "WEBDL"] # Fix for Radarr
+        - name: re_replace
+          args: ["WEBRIP.MHD.BDRip", "WEBRIP"] # Fix for Radarr
+        - name: append
+          args: "-XbytesV2"
+        - name: re_replace
+          args: ["\\.-XbytesV2", "-XbytesV2"] # Delete first dot
```

Comment:
Appending tracker name is not required and we are not doing it in other trackers. In Sonarr/Radarr you can configure tracker priority if you are trying to prefer xbytes2 results or something like that.

Reply:
Removed